scout/server/blueprints/diagnoses/controllers.py

```diff
@@ -28,3 +28,8 @@ def disease_terms(store):
 
     data = {"terms": store.disease_terms(filter_project=None)}
     return data
+
+
+def disease_terminology_count(store):
```

Comment:
Seems a bit like an overkill to have a function with 2 lines (could also be 1) of code, here. I know that this is the usual structure of the whole thing (views that collect data from controllers) but I think it would be more readable if you just did everything in the views, unless you had plans to add some other data in here in the future?

Reply:
I just figured I would follow the structure present for the other functions for readability

Reply:
It's OK, leave it like this

Reply:
The functionhas been moved to views